roles/logins/README.md

```diff
@@ -0,0 +1,85 @@
+# Cgroups limitation
+
+## PAM
+
+Systemd manages all users slices. These slices are located under main `user.slice`.
+Cgroups v2 can be managed separately, but users still cannot be moved into those
+because systemd already exclusively manage user slice.
```

Comment:
```suggestion
Cgroups v2 can be managed separately, but users cannot be moved into another slice hierarchy,
because `systemd` already exclusively manages user slices.
```